src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -130,6 +130,13 @@ private AddPropertyTransform<CommonCompilerOptions>[] KeyFileTransforms
                 project => "true",
                 project => project.GetProjectType() == ProjectType.Test);
 
+        private AddItemTransform<ProjectContext> AppConfigTransform =>
+            new AddItemTransform<ProjectContext>(
+                "None",
+                projectContext => "App.config",
+                projectContext => string.Empty,
+                projectContext => File.Exists(Path.Combine(projectContext.ProjectDirectory, "App.config")));
```

Comment:
I'm not 100% sure this will be enough on linux as this check might need to be case insensitive. This should suffice for now I believe since usually noone adds this file by hand